crates/noirc_evaluator/src/ssa.rs

```diff
@@ -50,6 +50,7 @@ pub(crate) fn optimize_into_acir(
             .inline_functions()
             .print(print_ssa_passes, "After Inlining:")
             .evaluate_assert_constant()?
+            .print(print_ssa_passes, "After asert cons:")
```

Comment:
typo: asert -> assert

Reply:
Ah, this should be removed entirely. It was only for testing